.goreleaser.yml

```
@@ -10,8 +10,8 @@ before:
     - 'go mod download'
 builds:
   -
-    # Binary naming only required for Terraform CLI 0.12
-    binary: '{{ .ProjectName }}_{{ .Version }}'
+    # _x4 suffix required for Terraform CLI < 0.12
+    binary: '{{ .ProjectName }}_{{ .Version }}_x4'
```

I would ensure `v` is included as well

```suggestion
    binary: '{{ .ProjectName }}_v{{ .Version }}_x4'
```

---

Aah yes, thanks.